mcv/nginx.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+from jinja2 import Template
+
+
+def nginx_reload():
+    return subprocess.call(["nginx -s reload"], shell=True)
+
+
+def rollout_servers(template_path, config_path, addrs):
+    """Render a set of server addresses into a Jinja2 template,
+    copy the output into the configuration path and reload
+    running nginx configuration
+
+    Template must expect `addrs` variable"""
+    tmpl = Template(open(template_path, "r").read())
```

Comment:
Usually you want this to be:

```python
with open(template_path, 'r'):
   ...
````

That way the file handle gets automatically closed properly and immediately.